static/styles/master.css

```diff
@@ -17,3 +17,8 @@ input {
   border: 1.4px solid #3e3e3e;
   border-radius: 7px;
 }
+
+
+.inptlab {
+  min-width: 100em;
```

Comment:
Seems that this class is making inputs too large, and they go out of the screen.

![image](https://user-images.githubusercontent.com/53529939/224197874-72d6f5b4-a861-4260-9f8a-db1f32697e00.png)